boards/shields/pca63566/doc/index.rst

```diff
@@ -0,0 +1,19 @@
+.. _pca63566:
+
+PCA63566 shield
+###############
+
+Overview
+********
+
+PCA63566 is an internal NRF testing shield.
```

Comment:
At the risk of repeating myself, why is this pushed upstream then? /cc @carlescufi

Reply:
See my comment [here](https://github.com/zephyrproject-rtos/zephyr/pull/71997#issuecomment-2084976538).